scripts/data-colection.js

```diff
@@ -19,19 +17,15 @@ const getBarStructure = (actor, location, modification) => {
 
 }
 
-const isPlutoniumNpc = (actor) => actor?.data?.flags?.core?.sourceId?.includes("plutonium");
-
-const isWarlockNpc = (actor) => actor?.data?.data?.details?.class?.name === "warlock";
-
 /**
- * So plutonium warlock npcs have this very beautiful bug, where they get generated being able to casts spells normally
+ * So warlock npcs have this very beautiful bug, where they get generated being able to casts spells normally
  * and this beautiful useless function is here to clear that useless info on them
  * It is not clean in any way, but it somewhat works
  *
  * @param spells - spell structure
  * @param actor - owner of the spells
  */
-const fixPlutoniumWarlockNpcs = (spells, actor) => {
+const fixPactMagic = (spells, actor) => {
     if (!isPlutoniumNpc(actor) || !isWarlockNpc(actor)) return;
```

Comment:
arent these 2 functions deleted now? (at least i can't find them)

Reply:
They are, but I think you are looking at an outdated version of that function.

Reply:
oh yes, and here i thought i understood how the whole PR stuff works :see_no_evil:  sorry!